Formula/qfs.rb

```diff
@@ -0,0 +1,37 @@
+class Qfs < Formula
+  desc "High-performance, fault-tolerant, distributed file system and its tools"
+  homepage "https://github.com/quantcast/qfs"
+  url "https://github.com/quantcast/qfs.git"
+  version "1.1.4-1"
```

Comment:
Done. We can wait for the new release, but I've used the hash for the latest master which should be a good stopgap for now until the next version is released.